modules/globebrowsing/shaders/globalchunkedlodpatch_vs.glsl

```diff
@@ -70,11 +79,24 @@ void main() {
     levelWeights = getLevelWeights(levelInterpolationParameter);
 
     // Get the height value
-    float height = getTileVertexHeight(in_uv, levelWeights);
+    float height = getTileHeight(in_uv, levelWeights);
 
     // Apply skirts
     height -= getTileVertexSkirtLength();
-    
+
+#if USE_ACCURATE_NORMALS
```

Comment:
I don't want to have to calculate tangents unless they are used. And they are only used if accurate normals are used.